src/theVoice/TheVoice.style.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+import { color, responsiveBreakpoints, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+import { Title } from '../title'
+
+export const StyledTheVoice = styled(Title)`
+  /* @TODO: Align scale and spec
+  /* 32px it's not part of current scale. 4 it could be calculated by the base unit space.s  */
+  --space-bottom: 32px;
+  /* 40px is an exception on font scale  */
+  --font-size-desktop: 40px;
+
+  margin: 0;
+  padding: ${space.xl} 0 var(--space-bottom);
+  white-space: pre-line;
+
+  ${normalizeHorizontally};
+
+  &.kirk-thevoice--inverted {
```

Comment:
Note: i used a class instead of the `isInverted` prop as `Title` doesn't handle this prop (TS was complaining)

Reply:
Some day refactor then.. should we create a ticket for that?

Reply:
We have these kind of things in lots of components. We'll see later as we already have more importants concerns IMHO

Reply:
Plus I don't think we want this prop in `Title`